files/default/neutron-ha-tool.py

```diff
@@ -982,6 +982,18 @@ def filter_routers(self, router_id_list):
         return router_id_list
 
 
+class WhiteListBasedRouterFilter(object):
```

Comment:
I think the `Based` is redundant here, and just adds a bit of noise.  Also, "whitelist" is a valid (single) word, so technically it's probably slightly more correct to write `Whitelist` here than `WhiteList`.  But I only mention that in an attempt to defeat all other contenders for the title of "Pedantry King of 2017".